### PR TITLE
Manual Upload

### PR DESCRIPTION
**Pull request by redpandarevolt**
Getting an error via GitHub desktop so the ReadMe was manually uploaded

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,194 @@
...
-* [GeeksForGeeks](https://www.geeksforgeeks.org/)
-* [EDUCBA Question Bank](https://www.educba.com/category/software-development/software-development-blog/top-interview-question/)[\*](#footnotes):
+<!-- Don't remove -->
+<a name="p2"/>
 
-<a name="resources-qs-breadth"/>
+### 2. Search in Rotated Sorted Array :bug:
 
-#### Breadth First (General Guides, Structured Courses)
+Source: <a href="https://www.youtube.com/watch?v=5BI0Rdm9Yhk">reference</a> :bug:
```
Comment (etcadinfinitum) [inline README.md]: Please include the LeetCode link; see writeups from other quarters for examples.

File: README.md [hunk 2]
```diff
@@ -1,116 +1,194 @@
...
+                if (nums[left] <= target && nums[mid] > target) right = mid - 1;
+                else left = mid + 1;
+            } else {
+                if (nums[mid] < target && nums[right] >= target) left = mid + 1;
+                else right = mid - 1;
+            }
+        }
+        return -1;
+    }
+};
```
Comment (etcadinfinitum) [inline README.md]: There are two halves of the README; the first half is for the problem statement (not including the solution code), and the second half is for the solution. This method would be written up in the Solution section, not the Problem section.

**Comment by redpandarevolt** [comment]
I don't use GitHub in this manner much.  Are you pretty much telling me to
make the changes directly to the read me in the 2020 readme link? I'm not
sure comfortable moving stuff deleting anything on here.

**Comment by etcadinfinitum** [comment]
Closing because #159 has been submitted in place of this PR.

**Pull request closed without merging**